frontend/containers/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -24,6 +25,71 @@ import DropdownNative from 'components/DropdownNative/DropdownNative'
 import Peer from 'components/Peer/Peer'
 import Toolbar from 'components/Toolbar/Toolbar'
 
+const FIELD_PUBLISH_SAVE_OPTIONS_LAST_USED = 'publishSaveOptionsLastUsed',
+  SAVE_AND_CONTINUE = 'Save and continue',
```

Comment:
I understand why we have the mode identifiers (e.g. `save`, `saveAndCreateNew`) defined as constants, since we'll use them on the interface and also save them to API. But do we need constants for the labels and for `publishSaveOptionsLastUsed`?

Reply:
What would our rules be for constants then? Should I continue separation or only use constants when two or more things use them?